client/my-sites/upgrades/checkout-thank-you/google-voucher/terms-and-conditions.jsx

```diff
@@ -2,41 +2,81 @@
  * External dependencies
  */
 import React from 'react';
-import i18n from 'i18n-calypso';
+import { localize } from 'i18n-calypso';
 
-const googleTermsAndConditions = () => {
-	return(
-		<div>
-			<h3>{ i18n.translate( 'Terms and conditions for this offer:' ) }</h3>
-			<p>{ i18n.translate( 'In the below terms, “AdWords” may mean AdWords or AdWords Express, as appropriate.' ) }</p>
-			<ol>
-				<li className="google-voucher__terms-and-conditions">{ i18n.translate( 'Offer available to customers with a billing address in the United States only. One promotional code per advertiser.' ) }</li>
+/*
+ * Constants
+ */
+const expirationDate = 'October 31, 2017';
```

Comment:
Just in case the English version:

<img src="https://cloud.githubusercontent.com/assets/77539/22122068/34a0cafc-de65-11e6-873d-a0f83c645efc.png" width="300px" />